custom/icds_reports/data_pull/sql_queries/vhsnd_monthly_report.sql

```diff
@@ -0,0 +1,15 @@
+SELECT state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    awc_site_code,
+    vhsnd_date_past_month
+FROM agg_awc_monthly INNER JOIN "ucr_icds-cas_static-vhnd_form_28e7fd58" vhnd_ucr ON agg_awc_monthly.awc_id=vhnd_ucr.awc_id AND vhnd_ucr.vhsnd_date_past_month>='{from_date}' AND vhnd_ucr.vhsnd_date_past_month<'{till_date}'
```

Comment:
`vhnd_ucr.vhsnd_date_past_month<'{from_date}::DATE + INTERVAL '1 month''`
or 
`vhnd_ucr.vhsnd_date_past_month<'({from_date}::DATE + INTERVAL '1 month')::DATE'`
Can this be updated to get rid of from_date and even the need to overrite `sql_query`?

Reply:
There might be a better way to add one month to the date, please do check if there is.

Reply:
changed it